security_monkey/exceptions.py

```diff
@@ -180,3 +180,26 @@ class UnableToAccessGoogleEmail(SecurityMonkeyException):
     def __init__(self):
         self.error_message = "Unable to fetch user e-mail. Please ensure your application has Google+ API access"
         app.logger.error(self)
+
+
+class SplunkAlerterFieldFormatError(SecurityMonkeyException):
```

Comment:
Would prefer that these be defined in a splunk specific plugin path.

Reply:
Agree, will move-it in next iteration